src/main/java/com/bank/View/BalanceView.java

```diff
@@ -13,6 +16,26 @@ public BalanceView(Account account) {
 
     @Override
     public void print(){
-        System.out.println("Your account balance is: " + account.getBalance() );
+        System.out.println("Your account balance is: " + account.getBalance());
+        System.out.println("Last transactions: ");
+        
+        /*
+        Old way
+        account.getTransactions().sort(Comparator.comparing(Transaction::getDate).reversed());
+        for (int i = 0; i < account.getTransactions().size(); i++) {
+            if( i  < 5) {
+               System.out.println(account.getTransactions().get(i));
+            }
+        }
+        */
+
+        account.getTransactions().stream()
+                .sorted(Comparator.comparing(Transaction::getDate).reversed())
+                .limit(5)
+                .forEach(transaction -> {
+                    String symbol = transaction.getType() == Transaction.Type.DEBIT ? "-": "+";
+                    System.out.printf("%s",symbol);
```

Comment:
nit: You can combine both printf together `System.out.printf("%s %.2f\n", symbol, transaction.getAmount())`